tests/main.test.ts

```diff
@@ -2445,4 +2471,81 @@ describe('zip-it-and-ship-it', () => {
     expect(result.stderr).not.toContain('Dynamic require of "path" is not supported')
     expect(result).not.toBeInstanceOf(Error)
   })
+
+  testMany(
+    'bundlers emit entry file with cjs extension when FF on',
```

Comment:
Nit for consistency.

```suggestion
    'Emits entry file with .cjs extension when `zisi_output_cjs_extension` flag is on',
```